### ISSUE TITLE
Ubuntu folder on file explore shows ridiculous file size

### ISSUE THREAD

**Issue by programmer1017** [microsoft/WSL#6054]
# Environment

```none
Windows build number: Win32NT, 10.0.20226.0, Microsoft Windows NT 10.0.20226.0
Your Distribution version: Release: 20.04
Whether the issue is on WSL 2 and/or WSL 1: Linux version 4.19.128-microsoft-standard (oe-user@oe-host) (gcc version 8.2.0 (GCC)) #1 SMP Tue Jun 23 12:58:10 UTC 2020
```

# Steps to reproduce

In file explore , at the Linux directory, Ubuntu folder, and I selected all in ubuntu folder and clicked 'properties' — I just wondering how much is the file size in it.
But surprisingly, it shows **128TB (140,738,768,950,860 bytes)** in it...
P.S: my C drive is about 100GB and even my HDD is about 1TB.... so I think 128TB is ridiculous. Can anyone explain why this happened? or is this just a bug?
![image](https://user-images.githubusercontent.com/66342883/95417785-dcc7c980-0970-11eb-828a-cfa26dc3ff6b.png)

**WSL logs**: 

#  Expected behavior

# Actual behavior

**Comment by licanhua** [comment]
yes, it's a known issue and a feature we didn't implement. also we don't have plan to fix it yet.

**Comment by shyney7** [comment]
Still an issue: 
![grafik](https://github.com/microsoft/WSL/assets/17785769/4af6d2aa-756f-4798-a89e-37e45d11b43f)
